app/scripts/controllers/faqGroup.ctrl.js

```diff
@@ -4,6 +4,42 @@ angular.module('yourStlCourts').controller('FaqGroupCtrl', function () {
   ctrl.faqName = camelize(ctrl.groupTitle);
   ctrl.autolinker = new Autolinker({stripPrefix:false});
 
+  for(var faqSection in ctrl.sourceData){
+    for(var faq in ctrl.sourceData[faqSection]){
+      var fillIn = ctrl.sourceData[faqSection][faq]["fill-in"];
+      switch(fillIn){
+        case "supportedMunicipalities":
+          var additionalData = findAdditionalData("supportedMunicipalities");
+          if (additionalData) {
+            var supportedMuniList = "Participating municipalities: ";
+            for (var municipalityCount = 0; municipalityCount < additionalData.length; municipalityCount++) {
+              if (municipalityCount > 0) {
+                supportedMuniList += ", ";
+              }
+              supportedMuniList += additionalData[municipalityCount].name;
+            }
+            ctrl.sourceData[faqSection][faq]["a"] += "<br><br>" + supportedMuniList;
+          }
+          break;
+      }
+    }
+  }
+
+  function findAdditionalData(dataLabel){
+    var foundData = null;
+    if (ctrl.additionalData){
+      for(var dataCount in ctrl.additionalData){
+        for(var label in ctrl.additionalData[dataCount]){
```

Comment:
You could probably replace these lines with a lodash _.find or something similar here for the nested loop. The nested loops are harder to read